### PR TITLE
added card to eventlistingstsx, added some more functionality to get …

### PR DESCRIPTION
**Pull request by rysmith504**
…req for events

### CODE REVIEW

**Review by dow2 — APPROVED** [review]
looks ok

File: server/routes/eventListingsRouter.ts [hunk 1]
```diff
@@ -10,13 +10,28 @@ const eventListingsRouter = Router();
 console.log('hello');
 
 eventListingsRouter.get('/list', (req, res) => {
-  console.log(req.query)
+  // console.log(req.query)
```
Comment (copelka) [inline server/routes/eventListingsRouter.ts:11]: do you need to leave these comments in?

File: server/routes/eventListingsRouter.ts [hunk 2]
```diff
@@ -10,13 +10,28 @@ const eventListingsRouter = Router();
 console.log('hello');
```
Comment (copelka) [inline server/routes/eventListingsRouter.ts]: remove all console logs before making a PR

File: server/routes/eventListingsRouter.ts [hunk 3]
```diff
@@ -10,13 +10,28 @@ const eventListingsRouter = Router();
...
+  axios.get(`https://app.ticketmaster.com/discovery/v2/events.json?size=10&keyword=${keyword}&apikey=${process.env.TICKETMASTER_API_KEY}`)
+    .then((responseObj) => {
+      const dates = responseObj.data._embedded.events.filter((event) => {
+        return event.dates.start
+      }).map(event => {
+        return ([event.id, event.dates.start])
+      })
+      console.log('DATES12' , dates);
+      console.log('RESPONSE OBJ GET', responseObj.data._embedded)
+      res.send({
```
Comment (copelka) [inline server/routes/eventListingsRouter.ts:27]: what are you sending?

**Pull request merged**